.github/workflows/test-and-deploy.yml

```diff
@@ -31,6 +30,11 @@ jobs:
 
       - run: bundle install --with development
       - run: bundle exec rake install
+
+      - name: Lint
+        run: make lint
+        if: ${{ !contains(["2.4"], matrix.ruby) }}
```

Comment:
Looks like this is being flagged as a [syntax error](https://github.com/twilio/twilio-ruby/actions/runs/1683882757) by GH actions